README.md

```diff
@@ -65,6 +65,13 @@ make install_emacs # Install emacs (>= 26.1) support (needs the eglot package)
 
```

Comment:
And there is still a reference to `make install_emacs` above, please grep for that and remove!

Reply:
(Weirdly, GitHub does not allow you to comment on code that was not directly modified.)

Reply:
It's been removed in 5c5ba3195d5a99